MixItUp.Desktop/Services/DesktopTelemetryService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.ApplicationInsights;
+using MixItUp.Base;
+using MixItUp.Base.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MixItUp.Desktop.Services
+{
+    public class DesktopTelemetryService : ITelemetryService
+    {
+        private TelemetryClient telemetryClient = new TelemetryClient();
+
+        public DesktopTelemetryService()
+        {
+            this.telemetryClient.Context.Cloud.RoleInstance = "MixItUpApp";
+            this.telemetryClient.Context.Cloud.RoleName = "MixItUpApp";
+            this.telemetryClient.Context.Session.Id = Guid.NewGuid().ToString();
+            this.telemetryClient.Context.Device.OperatingSystem = Environment.OSVersion.ToString();
+        }
+
+        public void TrackException(Exception ex)
+        {
+            if (!ChannelSession.Settings.OptOutTracking)
+            {
+                this.telemetryClient.TrackException(ex);
+            }
+        }
+
```

Comment:
We can track lots of different things. Events, Availability, Dependencies, Requests, Traces.  I just did these to as a quick example, but we could move the "Login" tracking in (SendLoginEvent) into this and just "TrackEvent".  Then the A.I. stuff lets us query and get charts and such.

Reply:
We don't want to "move" the log in event, but we can duplicate it. We need to track the log in events separately in the DB for the Feature Me feature.